mat64/io.go

```diff
@@ -0,0 +1,60 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mat64
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+var (
+	littleEndian  = binary.LittleEndian
+	bigEndian     = binary.BigEndian
+	defaultEndian = littleEndian
+)
+
+func writeBin(w io.Writer, data interface{}) error {
```

Comment:
Remove this function. It's one line, it's used three times, and the only thing it saves is typing "defaultEndian". By using binary.Write directly, it's clear that nothing tricky is happening.